tests/xtimer_now32_overflow/Makefile

```diff
@@ -7,6 +7,7 @@ DISABLE_MODULE += auto_init_xtimer
 
 # Boards that can't run only on xtimer or that always select ztimer
 BOARD_BLACKLIST += \
+                   esp32s2-wemos-mini \
```

Comment:
:confused: What is different on this board than on other ESP32-S2 boards?

Reply:
This should be `FEATURES_BLACKLIST += highlevel_stdio` but given that this will be removed soon anyway, I didn't want to test my luck.

Reply:
Ah, it is using `stdio_tinyusb_cdc_acm` which in turn depends on `ztimer_msec`.